devito/finite_differences/coefficients.py

```diff
@@ -224,6 +224,33 @@ def generate_subs(i):
         rules = {}
         for i in self.coefficients:
             rules.update(generate_subs(i))
+        return rules
+
+    def update_rules(self, obj):
+        """Update the specified rules to reflect staggering in an equation"""
```

Comment:
format for docstrings in devito:
```
"""
<line>.
"""
```
So each docstring takes at least three lines, and it always terminates with a full stop